SpatialGDK/Source/SpatialGDKServices/Public/LocalDeploymentManager.h

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+#pragma once
+
+#include "Async/Future.h"
+#include "CoreMinimal.h"
+#include "FileCache.h"
+#include "Modules/ModuleManager.h"
+#include "Templates/SharedPointer.h"
+
+DECLARE_LOG_CATEGORY_EXTERN(LogSpatialDeploymentManager, Log, All);
+
+class FJsonObject;
+
+class FLocalDeploymentManager
+{
+public:
+	FLocalDeploymentManager();
+
+	FSimpleMulticastDelegate OnSpatialShutdown;
```

Comment:
Do we normally do fields above methods or methods above fields?

Reply:
Should be below I believe. Will correct.